docs/concepts/function-components.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Function Components
+sidebar_label: Introduction
+description: Introduction to function components 
+---
+
+:::warning
+We're still working on function components and hooks. They're not quite ready to be used yet.
+If you'd like to help out, take a look at the [project board](https://github.com/yewstack/yew/projects/3) for a list of things that still need to be done.
+:::
+
+
+Function components are a simplified version of normal components.
+They consist of a single function that receives props and determines what should be rendered by returning `Html`.
+Basically, it's a component that's been reduced to just the `view` method.
+On its own that would be quite limiting because you can only create pure components, but that's where hooks come in.
+Hooks allow function components to use state and other Yew features without implementing the `Component` trait.
+
+## Example:
+ 
+```rust
+#[function_component(HelloWorld)]
+fn hello_world() -> Html {
+    html! {
+        { "Hello world" }
+    }
+}
+```
+
+## Creating function components
+
+The easiest way to create a function component is to add the [`#[function_component]`](function-components/attribute.md) attribute to a function.
+
+### Under the hood
+
+function components consists of two parts.
```

Comment:
```suggestion
Function components consists of two parts.
```
(Unless this was deliberate, but in that case we would have to change it in some other places)